client/my-sites/plugins/plugin-item/plugin-item.jsx

```diff
@@ -357,8 +311,4 @@ class PluginItem extends Component {
 	}
 }
 
-export default compose(
-	connect( null, { errorNotice } ),
-	localize,
-	withLocalizedMoment
-)( PluginItem );
+export default compose( localize, withLocalizedMoment )( PluginItem );
```

Comment:
nit: do we really need the `compose` here? Not related to your PR but maybe a good opportunity to clean it up :)

Reply:
Nope 😉  Good call, removed in https://github.com/Automattic/wp-calypso/pull/47964/commits/e6a086323278113f1acb335221b4da2c8226b351